modules/migrate-sdn-ovn.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//networking/ovn_kubernetes_network_provider/migrate-from-openshift-sdn.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="migrate-sdn-ovn-cli_{context}"]
+= Initiate migration using the ROSA CLI
+
+[WARNING]
+====
+You can only initiate migration on clusters that are version 4.15.22 and above or 4.16.24 and above.
+====
+
+To initiate the migration, run the following command:
+[source,terminal]
+----
+$ rosa edit cluster -c <cluster_id> <1>
+  --migrate-network-type OVNKubernetes
+  --ovn-internal-subnets <configuration> <2>
+----
+<1> Replace `<cluster_id>` with the ID of the cluster you want to migrate to the OVN-Kubernetes network plugin.
+<2> Optional: Users can create key-value pairs to configure internal subnets using any or all of the options `join, masquerade, transit` along with a single CIDR per option. For example, `--ovn-subnet-configuration="join=0.0.0.0/24,transit=0.0.0.0/24,masquerade=0.0.0.0/24"`.
+
+[IMPORTANT]
+====
+You cannot include the optional flag `--ovn-internal-subnets` in the command unless you define a value for the flag `--migrate-network-type`.
```

Comment:
`--network-type`

Sorry for the confusion earlier, I was confused as well since it was requested to change this back and forth